레이아웃 파일 일부 만들었습니다

JYPSloth opened the pull request:
레이아웃이 완벽하게 완성된 것은 아니지만 한 데까지는 공유가 필요할 것 같아서 pr합니다.
액티비티는 파일만 생성해 놓고 기능은 구현하지 않았습니다.
로그인 레이아웃(기존에 성민님이 만들어두신 것을 살짝만 수정), 학교 찾기 레이아웃, 회원 가입 레이아웃을 만들었습니다.
소셜 로그인과 학교 이메일 인증은 추후로 미룬다고 하여 레이아웃도 따로 만들지 않았습니다.
activity_univ_certify.xml 에서 학번을 spinner(일종의 콤보박스)를 통해 06~22학번 중 선택하도록 만들었습니다. 다만 UnivCertifyActivity에서 선택된 학번 정보를 처리하는 코드는 아직 구현하지 않았습니다.
activity_univ_certify.xml 에서 학교 검색창은 서버에서 학교 목록을 받아와서 보여주도록 해야 하는데, 그 부분은 구현되지 않았습니다.
실제 에브리타임 어플에서는 학교를 선택하고 나면 휴대폰 인증을 하게 하는데, 제가 이미 가입된 아이디가 있어서 휴대폰 인증 이후 회원가입 ui가 어떻게 생겼는지 확인을 하지 못해, 적당히 로그인 화면과 비슷하게 만들었습니다.
앞으로도 ui 담당과 통신 담당을 나눠서 작업하는 게 좋지 않을까 싶습니다.

JYPSloth committed:
refactor(frontend): LoginActivity.kt 추가, activity_login.xml 추가
레이아웃 파일만 수정하고 기능은 손대지 않았습니다.

JYPSloth committed:
refactor(frontend): LoginActivity.kt 추가, activity_login.xml 추가
레이아웃 파일만 수정하고 기능은 손대지 않았습니다.

JYPSloth committed:
학교 선택 레이아웃 추가(미완성)

JYPSloth committed:
회원가입 레이아웃

JuTaK97 commented:
앗 pull request 이전 거를 보고 있었네요 다시 확인해 보겠습니다!

JYPSloth committed:
login activity에 mainactivity가 binding되어 있어서 수정함.
margin과 padding 값 조정

JuTaK97 committed:
signup 일부 로직 추가

JuTaK97 committed:
login, signup

JuTaK97 commented:
로그인, 회원가입 부분 어느 정도 만들었는데 서버랑 통신 실제로 해 보면서 디버깅을 좀 해야될 것 같아요.

JuTaK97 committed:
login/signup

JuTaK97 commented:
로그인, 회원가입 기능 만들었습니다.

남은 일은, 
1. 잘못 입력했을 때 적절한 에러 메시지 띄워주는 것
2. nickname 중복 확인 버튼 만들기 및 처리
3. name 입력하는 칸 만들기
4. 디버깅 하기
정도가 되겠네요. 일단 apk 공유하고 버그를 제보 받도록 하겠습니다.

JYPSloth committed:
다크모드에 대비해서 로고 이미지를 배경투명화된 것으로 바꿨습니다.(로그인 레이아웃과 회원가입 레이아웃 수정)
회원가입 레이아웃에 이름 입력란을 추가했고, 중복확인 버튼이 차지하는 공간을 줄이려고 framelayout을 이용해 입력란 위에 중복확인 버튼이 위치하도록 수정하였습니다.

JYPSloth committed:
회원가입 레이아웃에 이름 입력란 추가하는 것을 깜박해 추가했고,하드코딩된 텍스트를 string으로 추출하였습니다. 레이아웃과 strings.xml 파일 외에는 손댄 것 없습니다.

JuTaK97 committed:
에러 메시지 수정 및 중복확인 수정

JuTaK97 committed:
Merge branch 'feature/layouts' of https://github.com/wafflestudio19-5/team11-android into feature/layouts

JuTaK97 committed:
에러 메시지 추가, nickname 관련 로직 추가
